README.md

```diff
@@ -2,10 +2,20 @@
 
 ScalarDL is a scalable and practical Byzantine fault detection middleware for transactional database systems that achieves correctness, scalability, and database agnosticism.
 
-## Docs
-* [ScalarDL Documentation](https://scalardl.scalar-labs.com/docs/latest/)
-* [ScalarDL Technical Overview](https://speakerdeck.com/scalar/scalar-dl-technical-overview)
-* [ScalarDL Research Paper](https://dl.acm.org/doi/abs/10.14778/3523210.3523212) [VLDB'22]
+To learn more, see [ScalarDL Overview](https://scalardl.scalar-labs.com/docs/latest/overview).
+
+## Quickstart
+
+1. Start running contracts.
+   - **Get started with ScalarDL Ledger:** To set up a basic application and run contracts by using ScalarDL Ledger, see [Get Started with ScalarDL Ledger](https://scalardl.scalar-labs.com/docs/latest/getting-started).
+   - **Get started with ScalarDL Auditor (for enterprise customers):** To set up a basic application and run contracts by using ScalarDL Ledger and then use ScalarDL Auditor to detect whether those contracts have been tampered with, see [Getting Started with ScalarDL Auditor](https://scalardl.scalar-labs.com/docs/latest/getting-started-auditor).
+2. See how to set up [a simple bank account application that uses ScalarDL](https://scalardl.scalar-labs.com/docs/latest/applications/simple-bank-account).
+3. Learn about the [configurations for ScalarDL](https://scalardl.scalar-labs.com/docs/latest/configurations)
+4. Learn about features, such as:
+   - [Authentication](https://scalardl.scalar-labs.com/docs/latest/authentication)
```

Comment:
I considered removing this since Authentication might not be a big feature and an interesting topic for a quick start. But, for now, it appears acceptable.

Reply:
Yeah... I was having a little trouble figuring out which docs had features that would be good for beginners. If we have any other docs or when we create more in the future, let's be sure to add them here for visibility.

Reply:
If they don't have to be features, the design doc and implementation doc should be useful for users.
The same applies to the ScalarDB README.

Reply:
Thanks for the suggestion! I've added those two docs here and will do similar in the ScalarDB README.